<?xml version="1.0"?>

<launch>
  <node name="filter_lidar" pkg="igvc_perception" type="filter_lidar" output="screen">
      <param name="input_topic" value="/velodyne_points" />
      <param name="output_topic" value="/scan/pointcloud"/>

      <!-- Minimum distance to accept scans from relative to the lidar frame (m.) -->
      <param name="x_front" type="double" value="1.0"/>
      <param name="x_back" type="double" value="1.0"/>
      <param name="y_left" type="double" value="1.0"/>
      <param name="y_right" type="double" value="1.0"/>
      <param name="z_up" type="double" value="1.0"/>
      <param name="z_down" type="double" value="1.0"/>

      <!-- Maximum distance to accept scans from relative to the lidar frame (m.) -->
      <param name="maximum_distance" value="15"/>
    </node>
</launch>
